recipe/patch_yaml/arm-variant.yaml

```diff
@@ -0,0 +1,70 @@
+# https://github.com/conda-forge/arm-variant-feedstock/issues/6 arm-variant should be a run
+# dependency not a run constrained. We want to move the dependency from run_constrained to
+# run, but the patcher raises an error if we do that, so we just add the dependency to run.
+if:
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  has_constrains: "arm-variant * sbsa"
+  timestamp_lt: 1748544655000
+then:
+  - add_depends: "arm-variant * sbsa"
+  # - remove_constrains: "arm-variant * sbsa"
+---
+if:
+  subdir_in:
+    - linux-aarch64
+    - noarch
+  has_constrains: "arm-variant * tegra"
+  timestamp_lt: 1748544655000
+then:
+  - add_depends: "arm-variant * tegra"
+  # - remove_constrains: "arm-variant * tegra"
```

Comment:
Are there any cases that use Tegra already? If so, think we can build a small set of packages here. If not, think we can drop this

Guessing we are already in the latter case, but would be interested to learn if there are some packages already we need to pay attention to